data/remnant/remnant 2 cognizance.txt

```diff
@@ -712,7 +712,7 @@ mission "Remnant: Cognizance 20"
 			`	"Captain, was there anything unusual when you arrived there?"`
 			choice
 				`	"Yes, sensors reported life signs."`
-				`	"I don't remember. Did you notice something?"`
+				`	"I do not remember. Did you notice something?"`
```

Comment:
This and the next two from the player all come across as significantly stilted and awkward to me; I think a more extensive change may be best.

I'll try to be a bit more concrete about suggestions after work.

Reply:
Simple change in wording makes it much better: Instead of "I do not remember", swap to "I forget"